counsel.el

```diff
@@ -6356,12 +6361,13 @@ text with FACE."
                       'font-lock-warning-face)
           (propertize text 'face face)))
 
-(defun counsel--compile-get-make-targets (srcdir &optional blddir)
-  "Return a list of Make targets for a given SRCDIR/BLDDIR combination.
+(defun counsel--compile-get-make-targets (probe-fn srcdir &optional blddir)
+  "Return propertied make targets returned by `PROBE-FN' in `SRCDIR'.
```

Comment:
propertied -> propertized

Also, metavariables here and elsewhere should not be surrounded by backtick/apostrophe; see [`(info "(elisp) Documentation Tips")`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Documentation-Tips.html).